src/btree/bt_vrfy_dsk.c

```diff
@@ -293,7 +293,7 @@ __verify_dsk_row(
 		case WT_CELL_ADDR_LEAF_NO:
 		case WT_CELL_KEY_OVFL:
 		case WT_CELL_VALUE_OVFL:
-			if (!bm->addr_valid(bm,
+			if (bm->addr_invalid(bm,
 			    session, unpack->data, unpack->size))
```

Comment:
It seems wrong for `__wt_vunpack_uint` to return `ENOMEM`.  That makes sense when packing an integer if the supplied buffer isn't big enough, but not for unpacking.  If we change `__wt_vunpack_uint` to return `EINVAL` in that case, are there any other cases you're aware of where this code has to handle any error return other than `EINVAL`?

My feeling in general is that we should check for specific returns (which if we change `__wt_vunpack_uint` should just be `EINVAL`) and otherwise use `WT_ERR` / `WT_RET` to handle errors without building in assumptions about what the called functions return.